extensions/adobe/b2b/marketo/marketo-web-url.schema.json

```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
-  "description": "Use this mixin for web url which does not confine with URI format.",
+  "description": "Use this field group for web url which does not confine with URI format.",
```

Comment:
What does this field group capture? It's not clear from the description. 

"Captures web url, which need not be in URI format"

?